CHANGELOG.next.asciidoc

```diff
@@ -46,6 +46,20 @@ https://github.com/elastic/beats/compare/v8.2.0\...main[Check the HEAD diff]
 *Filebeat*
 
 - Fix a hang on `apt-get update` stage in packaging. {pull}32580[32580]
+- gcp-pubsub input: Restart Pub/Sub client on all errors. {issue}32550[32550] {pull}32712[32712]
```

Comment:
@ChrsMark any idea why the changelog shows me much more than just my new changelog line at 55?